ui/selectmenu.js

```diff
@@ -498,6 +501,7 @@ return $.widget( "ui.selectmenu", {
 		// Change native select element
 		this.element[ 0 ].selectedIndex = item.index;
 		this.buttonItem.replaceWith( this.buttonItem = this._renderButtonItem( item ) );
+		this._addClass( this.buttonItem, "ui-selectmenu-text" );
```

Comment:
I'm not talking about a separate method. `_renderButtonItem` already exists, and for all three calls you added this `_addClass` call. You can move the `_addClass` call into the `_renderButtonItem` and get rid of two duplicates.